hazelcast/src/main/java/com/hazelcast/util/executor/StripedExecutor.java

```diff
@@ -149,7 +149,15 @@ private Worker getWorker(Runnable command) {
         return workers[index];
     }
 
-    // used in tests.
+    public List<BlockingQueue<Runnable>> getWorkQueues() {
```

Comment:
I'm not. The code should look familiar to you, I took the whole sampling approach from the `OverloadedConnectionsPlugin`:
```java
int sampleCount = min(samples, packets.size());
```
etc.